docs/cli_installation.md

```diff
@@ -10,12 +10,6 @@ You can download the latest Argo CD version from [the latest release page of thi
 pacman -S argocd
 ```
 
-### Homebrew
```

Comment:
Most linux users don't prefer using brew way to install packages. Hence, I have moved this instruction to Mac installation steps.
Hope that makes sense @cardoe

Reply:
"Most" but not "Every" :)